src/hooks.test.tsx

```diff
@@ -124,5 +134,121 @@ describe('multicall hooks', () => {
         expect(container?.textContent).toBe('true')
       })
     })
+
+    describe('utilizes correct blocksPerFetch values from defaultListenerOptions in store', () => {
+      it('utilizes blocksPerFetch configured in defaultListenerOptions in store', () => {
+        const callA = { address: 'a', callData: '' }
+        updateCallResult(callA, '0xa')
+
+        store.dispatch(
+          actions.updateDefaultListenerOptions({
+            chainId: 1,
+            listenerOptions: {
+              blocksPerFetch: 10,
+            },
+          })
+        )
+
+        const mockContext = {
+          reducerPath: 'multicall',
+          actions: {
+            addMulticallListeners: jest
+              .fn()
+              .mockImplementation((arg: Object) => ({ type: 'multicall/addMulticallListeners', payload: arg })),
+            removeMulticallListeners: jest
+              .fn()
+              .mockImplementation((arg: Object) => ({ type: 'multicall/removeMulticallListeners', payload: arg })),
+          },
+        }
+
+        act(() => {
+          render(
+            <Provider store={store}>
+              <Caller calls={[callA]} multicallContext={mockContext} />
+            </Provider>,
+            container
+          )
+        })
+
+        expect(mockContext.actions.addMulticallListeners).toHaveBeenCalledWith({
+          chainId: 1,
+          calls: [{ address: 'a', callData: '' }],
```

Comment:
nit: re-use the initial argument variable above

Reply:
(same below)

Reply:
Sounds good done